processor/transform/reusable-content.js

```diff
@@ -5,23 +5,20 @@ export const type = 'reusable-content';
 const regexp = /^\s*<(?<tag>[A-Z]\S+)\s*\/>\s*$/;
 
 const reusableContentTransformer = function () {
-  const { tags, disabled } = this.data('reusableContent');
+  const { tags, disabled, writeTags = true } = this.data('reusableContent');
```

Comment:
I sort of think this should use a different option, maybe `wrapReusableContent`??

The `writeTags` option was added for exporting docs. Maybe it should have been named better, like `serializeReusableContent` or something??

Reply:
sounds good! i had just seen `writeTags` being used to send RC content in a form that the TOC would also be able to pick up, so it seemed reasonable. having a new option is probably a good idea to separate the concerns!

Reply:
also, reusable content gets spat out in a container `div` in the hub right now, so unwrapping it also means less room for weird styling and selecting issues

Reply:
Yea, I def agree we should not be rendering it that way.

Reply:
gonna change these two options to `wrap` and `serialize` too be less clunky, since they're already in a `reusableContent` obj, what'd'you think

Reply:
Excellent